mods/d2k/tilesets/arrakis.yaml

```diff
@@ -6731,3 +6731,1537 @@ Templates:
+			Segment:
+				Start: RockSmooth.U
+				Inner: RockSmooth
+				End: RockSmooth.U
+				Points: 2,2, 2,1, 1,1, 1,0
+		MultiBrush@29:
+			Template: 46
+			Segment:
+				Start: RockSmooth.R
+				Inner: RockSmooth
+				End: RockSmooth.R
+				Points: 0,1, 1,1, 1,2, 2,2
+		MultiBrush@30:
+			Template: 45
+			Segment:
+				Start: RockSmooth.R
+				Inner: RockSmooth
+				End: RockSmooth.R
+				Points: 0,2, 1,2, 1,1, 2,1
+		MultiBrush@31:
+			Template: 44
+			Segment:
+				Start: RockSmooth.L
+				Inner: RockSmooth
+				End: RockSmooth.L
+				Points: 2,0, 1,0, 1,1, 0,1
+		MultiBrush@32:
+			Template: 21
+			Segment:
+				Start: RockSmooth.L
+				Inner: RockSmooth
+				End: RockSmooth.L
+				Points: 2,1, 1,1, 1,0, 0,0
+		MultiBrush@33:
+			Template: 36
+			Segment:
+				Start: RockSmooth.D
+				Inner: RockSmooth
+				End: RockSmooth.D
+				Points: 1,0, 1,1, 0,1, 0,2
+		MultiBrush@34:
+			Template: 32
+			Segment:
+				Start: RockSmooth.D
+				Inner: RockSmooth
+				End: RockSmooth.D
+				Points: 0,0, 0,1, 1,1, 1,2
+		MultiBrush@35: # SandRockCliffs
+			Template: 108
+			Segment:
+				Start: SandRockCliff.R
+				Inner: SandRockCliff
+				End: SandRockCliff.R
+				Points: 0,1, 1,1
+		MultiBrush@36:
+			Template: 187
+			Segment:
+				Start: SandRockCliff.R
+				Inner: SandRockCliff
+				End: SandRockCliff.R
+				Points: 0,1, 1,1, 2,1
+		MultiBrush@37:
+			Template: 188
+			Segment:
+				Start: SandRockCliff.R
+				Inner: SandRockCliff
+				End: SandRockCliff.R
+				Points: 0,1, 1,1, 2,1
+		MultiBrush@38:
+			Template: 182
+			Segment:
+				Start: SandRockCliff.R
+				Inner: SandRockCliff
+				End: SandRockCliff.R
+				Points: 0,1, 1,1, 2,1
+		MultiBrush@39:
+			Template: 200
+			Segment:
+				Start: SandRockCliff.R
+				Inner: SandRockCliff
+				End: SandRockCliff.R
+				Points: 0,1, 1,1, 1,2, 2,2
+		MultiBrush@40:
+			Template: 206
+			Segment:
+				Start: SandRockCliff.R
+				Inner: SandRockCliff
+				End: SandRockCliff.R
+				Points: 0,2, 1,2, 1,1, 2,1
+		MultiBrush@41: # RS DOWN
+			Template: 258
+			Segment:
+				Start: SandRockCliff.L
+				Inner: SandRockCliff
+				End: SandRockCliff.L
+				Points: 1,1, 0,1
+		MultiBrush@42:
+			Template: 194
+			Segment:
+				Start: SandRockCliff.L
+				Inner: SandRockCliff
+				End: SandRockCliff.L
+				Points: 2,2, 1,2, 1,1, 0,1
+		MultiBrush@43:
+			Template: 196
+			Segment:
+				Start: SandRockCliff.L
+				Inner: SandRockCliff
+				End: SandRockCliff.L
+				Points: 2,1, 1,1, 0,1
+		MultiBrush@44:
+			Template: 255
+			Segment:
+				Start: SandRockCliff.L
+				Inner: SandRockCliff
+				End: SandRockCliff.L
+				Points: 2,1, 1,1, 0,1
+		MultiBrush@45:
+			Template: 256
+			Segment:
+				Start: SandRockCliff.L
+				Inner: SandRockCliff
+				End: SandRockCliff.L
+				Points: 2,1, 1,1, 0,1
+		MultiBrush@46:
+			Template: 197
+			Segment:
+				Start: SandRockCliff.L
+				Inner: SandRockCliff
+				End: SandRockCliff.L
+				Points: 2,1, 1,1, 1,2, 0,2
+		MultiBrush@47: #TODO this should have lower weight
```

Comment:
You might want to watch #21924 - that'll make weights respected.

(Even without that PR merged, I think you can still specify a weight in anticipation without it crashing anything.)

Reply:
Do I need to set up weight for all multibrushes or just one with lower weights? 
Or what is the default weight if weight is not set?